packages/web-components/fast-element/src/components/element-controller.ts

```diff
@@ -322,10 +322,18 @@ export class ElementController<TElement extends HTMLElement = HTMLElement>
             return;
         }
 
-        if (this.needsInitialization) {
-            this.finishInitialization();
-        } else if (this.view !== null) {
-            this.view.bind(this.source);
```

Comment:
For the previous issue, is it because `this.view.bind(this.source)` is called before `key.connectedCallback && key.connectedCallback(this);`? Silly question: is it possible `this.needsInitialization` is false, but `this.boundObservables !== null`?

Reply:
Sorry, which previous issue?

No, that case is not possible. Both of those conditions can only be true before / during the first connection of the controller. However, it is possible that boundObservables is null while needsInitialization is true.

Reply:
Sorry, I mean the "current issue". I got the answer now, thx :)